pymc/distributions/multivariate.py

```diff
@@ -1649,20 +1649,20 @@ class MatrixNormal(Continuous):
 
     Parameters
     ----------
-    mu: array
+    mu : tensor_like of float
         Array of means. Must be broadcastable with the random variable X such
         that the shape of mu + X is (m,n).
-    rowcov: mxm array
+    rowcov : mxm tensor_like of float, optional
```

Comment:
Also, can you update this to be 

```suggestion
    rowcov : (M, M) tensor_like of float, optional
```

and the same for the other similar cases. I know this didn't appear in the guide, but doing that we'd be following matplotlib's convention instead of creating a new one. Ref: https://matplotlib.org/stable/devel/documenting_mpl.html#parameter-type-descriptions. It is also not part of the numpydoc description but it also seems to be the convention used by numpy itself, see https://numpy.org/doc/stable/reference/generated/numpy.linalg.eig.html for example

Note: similar comment for @Sayam753 and @purna135. Not sure if it should be `(M, M)` or `(..., M, M)`, also, if it is M,M but will be updated to ..., M, M please keep M, M for now but remember to update it once it is improved 🚀

Reply:
Only 2 dimensions are supported for rowcov currently i.e. (M, M). @purna135 will work on adding batch support  post GSoC mid-evaluations to make it `(..., M, M)`.

Reply:
I'll keep it bidimensional in the documentation for now and @purna135 can change it when the feature gets added. :)